.vortex/docs/questions.md

```
@@ -0,0 +1,154 @@
+We need a section on upgrading, not sure what to put it, but that section should describe the upgrade process for existing projects.
```

markdownlint-cli2 (0.17.2)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)